api/api.go

```diff
@@ -102,6 +102,7 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 		r.With(sharedLimiter).With(api.requireAdminCredentials).Post("/invite", api.Invite)
 		r.With(sharedLimiter).With(api.verifyCaptcha).Post("/signup", api.Signup)
 		r.With(sharedLimiter).With(api.verifyCaptcha).With(api.requireEmailProvider).Post("/recover", api.Recover)
+		r.With(sharedLimiter).With(api.verifyCaptcha).With(api.requireEmailProvider).Post("/resend_confirmation", api.SendConfirmation)
```

Comment:
Is it possible to move this to `recover`? I feel like there's the SMS OTP use case that can be covered here as well. I'm not keen on adding just another API endpoint.

If we are getting to this, I'd also build database level rate limiting per email address/phone number so that you can only request this every 30 seconds for the same email address/phone number, as this is easy to abuse.

Reply:
Actually probably for this specific case this should maybe be a special case for sign up?

Reply:
> so that you can only request this every 30 seconds for the same email address/phone number

we have this already, the config setting is called `SMTP_MAX_FREQUENCY` and the default is set to 60s